src/components/ValidatorsList.react.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import Store from '../store'
+
+import ValidatorsActions from '../actions/validators'
+
+class ValidatorsList extends Component {
+  componentWillMount() {
+    Store.dispatch(ValidatorsActions.findAll())
+  }
+
+  render() {
+    const validators = this.props.validators.list
+    return (
+      <div>
+        <p>Validators:</p>
+        { validators.length === 0 ? (
+            <em>No validators</em>
+          ) : (
+            <ul>{this._buildValidatorsList(validators)}</ul>
+          )
+        }
+      </div>
+    )
+  }
+
+  _buildValidatorsList(validators) {
+    return validators.map((validator, index) => {
+      return (
+        <li key={index}>
```

Comment:
If possible, try to avoid using `index` as key, for performance reasons. Maybe validators have a unique address or id? I don't recall the model TBH.

Reply:
The validator is unique, it's an address. I will use that instead.

Reply:
Interesting! Here's an article about it for the record: https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318